### DISCUSSION TITLE
Support for ssh_config_file for eos devices

### DISCUSSION THREAD

**Discussion by impretzel** [napalm-automation/napalm#1797] [Q&A]
Hello,

Thank you for bringing SSH transport support for eos on napalm 4.0.0.

We have some Arista switches behind the SSH proxy server. May I know if napalm support the option_args `ssh_config_file` for eos devices?

Thanks.

**Comment by fischa** [comment]
Hi @impretzel ,
there is #1750 that should fix it when done. I would assume you’re running into the same issue with ssh config not being honoured? In my particular case for devices behind a jump box?

**Reply by afischer1-godaddy** [reply]
@impretzel this is working now when you use the development branch of napalm till the next release is build.
